tagr/storage/local.py

```diff
@@ -50,6 +50,23 @@ def dump_pickle(self, pickle_object, proj, experiment, tag, filename):
             "{}/{}/{}/{}.pkl".format(proj, experiment, tag, filename), "wb"
         ))
     
+    def list(self, proj, experiment, tag):
```

Comment:
I see we have a method named `list()` in `artifacts.py`. lets make this naming explicit. rename to private function